README.md

```diff
@@ -5,7 +5,7 @@
 
 # if me
 
-if me is a community for mental health experiences that encourages people to
+[_if me_](https://www.if-me.org/) is a community for mental health experiences that encourages people to
```

Comment:
Please keep lines to 80 characters. In markdown, some exceptions are allowed for things like URLs or where a line break could cause difficulties with the flow of reading, however, most of the time you should introduce a new line.

Reply:
Done!